easybuild/easyblocks/f/fftw.py

```diff
@@ -116,6 +116,11 @@ def __init__(self, *args, **kwargs):
                     self.log.info("Enabling use of %s (should be supported based on CPU features)", flag.upper())
                     setattr(self, flag, True)
 
+        # Auto-disable quad-precision on ARM and POWER, as it is unsupported
+        if self.cfg['with_quad_prec'] and arch in [AARCH32, AARCH64, POWER]:
```

Comment:
@boegel I explicitly mentioned that this was **not** tested.  And yes, you are right w.r.t. `cpu_arch`.